Fix quotes in vim command

galou opened the pull request.

taketwo commented:
Hi. The `escape()` function is supposed to take care of the quotes, so the bug is there. In fact, I believe I have a fix for this in another branch (which I was going to merge into master soon). In the meantime, maybe you can try to cherry-pick https://github.com/taketwo/vim-ros/commit/39633bd8080ea8600408d7ddd76e555b59b4d556 and see if it helps?

galou commented:
Hi, sorry, I had supposed that the escape function is doing this but looked too quickly at it and didn't seen the `if isinstance(obj, str)` clause. I reverted my changes, applied the proposed fix. 39633bd fixes the issue with quotes. I updated my branch.
